Drive/rover_drive_system.hpp

```diff
@@ -379,6 +402,7 @@ class RoverDriveSystem
     }
   };
 
+  int heartbeatCount = 0;
```

Comment:
this should be `heartbeat_count_` to follow [syntax style guide](https://google.github.io/styleguide/cppguide.html#Variable_Names)